src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1439,7 +1439,13 @@ internal enum ErrorCode
         ERR_DelegateRefMismatch = 8189,
         #endregion stragglers for C# 7
 
-        // Available  = 8190-8195
+        #region diagnostics for parse options
+        WRN_BadSourceCodeKind = 8190,
```

Comment:
Because the Obsolete attribute is a warning. We can turn it into an error when the obsolete attribute changes as well:
[https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/Portable/SourceCodeKind.cs#L28](https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/Portable/SourceCodeKind.cs#L28)

---
In reply to: [101839466](https://github.com/dotnet/roslyn/pull/17113#discussion_r101839466) [](ancestors = 101839466)